### PR TITLE
feat(header): create hamburger menu for mobile responsive header

### LINKED ISSUE #21: Create Hamburger menu for header mobile responsive

### PR DESCRIPTION
**Pull request by sahanperera00**
fixes #21

### CODE REVIEW

**Review by senuravihanjayadeva — APPROVED** [review]

**Comment by uihalagedara** [comment]
upload a screenshot of the implemented part and try to use webp images

File: src/components/client/common/header/index.jsx [hunk 1]
```diff
@@ -1,58 +1,63 @@
-import React from 'react'
 import './header.css'
 import logo from '../../../../assets/logo/FCSCMedia.png'
```
Comment (uihalagedara) [inline src/components/client/common/header/index.jsx]: try to use webp images

File: src/components/client/common/header/index.jsx [hunk 2]
```diff
@@ -1,58 +1,63 @@
...
-            href="/"
-            className="d-flex align-items-center mb-3 mb-md-0 me-md-auto text-decoration-none"
-          >
+    <header>
+      <nav class="navBarCont navbar navbar-expand-lg bg-body-tertiary">
+        <div class="navBarInnerCont container-fluid">
+          <a class="navbar-brand" href="#">
             <div className="logoDiv d-flex justify-content-center align-items-center">
               <div>
                 <img src={logo} alt="" width="90" />
```
Comment (uihalagedara) [inline src/components/client/common/header/index.jsx:13]: add a description to the "alt"

**Comment by sahanperera00** [comment]
> upload a screenshot of the implemented part and try to use webp images

image
image
image

File: src/components/client/common/header/index.jsx (hunk 1 above)
Comment (sahanperera00) [reply src/components/client/common/header/index.jsx]: image

**Review by uihalagedara — APPROVED** [review]

**Pull request merged**